packages/@okta/vuepress-theme-prose/package.json

```diff
@@ -25,8 +25,11 @@
   "dependencies": {
     "@okta/odyssey": "^0.0.1",
     "axios": "^0.21.0",
+    "cookies.js": "^2.1.15",
     "include-media": "^1.4.9",
     "moment": "^2.24.0",
+    "moment-timezone": "^0.5.32",
+    "query-string": "^6.13.8",
```

Comment:
This adds a bunch of libs but wanted to keep code as is for now, should def revisit for phase 2, including maybe replacing `moment` and `moment-timezone` with something lighter like `date-fns`